zulipterminal/ui_tools/boxes.py

```diff
@@ -1567,7 +1569,9 @@ def main_view(self) -> List[Any]:
             text_keys = ("author", "star", "time", "status")
             text: TextType = {key: (None, " ") for key in text_keys}
 
-            if any(different[key] for key in ("recipients", "author", "24h")):
+            if any(
+                different[key] for key in ("recipients", "author", "24h", "author_id")
+            ):
```

Comment:
This is now new behavior, so while it passes existing tests, it would be good to extend or add a test to illustrate this, and include it in this commit.